drivers/android-driver/build.gradle

```diff
@@ -6,11 +6,6 @@ archivesBaseName = 'sqldelight-android-driver'
 android {
   compileSdkVersion versions.compileSdk
 
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_7
-    targetCompatibility JavaVersion.VERSION_1_7
-  }
```

Comment:
You still want all these, just specified as 8. Otherwise if I'm building with JDK 11 or 12 (my default) you'll get bytecode targeting 12.

Reply:
I wish there was an easy way specify these once just as easy as Alec did above for kotlin tasks.
As far as I've managed to google this topic, it requires some gradle-fu to achieve this.

Reply:
It's not too dissimilar from what he posted. Something like

```groovy
plugins.withId("com.android.library") {
  extensions.getByName("android").compileOptions {
    // stuff
  }
}
```

Reply:
I need to try that! We have 10x modules and specifying options manually for each of them is rather daunting task (it can be copy pasted, but still)...

Reply:
I would recommend viewing https://events.google.com/io/schedule/events/5459b7ee-d067-4576-9e12-91546cd5bcd0 when it's available. It will have some best-practice for managing configuration of 10+ module projects where copy/paste becomes unviable.

Reply:
Great, will do, thanks!

Reply:
do i need to do this for only android modules?

Reply:
No, both.

JVM would be something like
```groovy
tasks.withType(JavaCompile) {
  sourceCompatibility = ...
  targetCompatibilty = ..
}
```